src/web/handlers/GestureHandler.ts

```diff
@@ -288,33 +288,20 @@ export default abstract class GestureHandler implements IGestureHandler {
     if (this.pointerType === PointerType.TOUCH) {
       GestureHandlerOrchestrator.getInstance().cancelMouseAndPenGestures(this);
     }
-
-    if (this.config.needsPointerData) {
-      this.sendTouchEvent(event);
-    }
```

Comment:
Why aren't we sending events in onPointerDown anymore?

Reply:
What I wanted to achieve was to send touch event after gesture had opportunity to begin. All `super.onPointerDown` are called at the beginning of overriden methods because parent class does some additional steps (like adding handler to `orchestrator`). The easiest way to get this flow was to remove sending touch event in parent class and call it at the end of child's `onPointerDown`.

Reply:
In this case, it may be worth to consider refactoring it so that `onPointerDown` in the base `GestureHandler` class looks something like this:

```
onPointerDown(event) {
  // do the common stuff

  this.handleDownEvent(event);
  this.tryToSendTouchEvent(event);
}
```

and move the handler specific stuff to `handleDownEvent` method.

It's outside the scope of this PR though, but if you agree a TODO in code would be great so it's not lost to time 😅.

Reply:
Sounds good, left a `TODO` in [f2ac147](https://github.com/software-mansion/react-native-gesture-handler/pull/2923/commits/f2ac1471b9c9c25396246d8db0debbb4787358c7)